bots/tests-scan

```diff
@@ -163,8 +183,22 @@ def tests_human(priority, name, revision, ref, context, base, repo, bots_ref):
         bots_ref=bots_ref and (" [bots@%s]" % bots_ref) or "",
     )
 
+def queue_test(priority, name, revision, ref, context, base, repo, bots_ref, queue, options):
```

Comment:
Matter of preference, but I don't think it makes too much sense to split this into two functions.